src/components/GenerateGifForm.js

```diff
@@ -21,15 +28,50 @@ class GenerateGifForm extends Component {
       case 'fontColor':
         this.props.updateTextColor(evt.target.value);
         break;
+
       case 'name':
         this.props.updateGIFFileName(evt.target.value);
         break;
+
+      case 'placement':
+        let values = evt.target.value.split('-');
```

Comment:
You can clean this up by using array destructuring:

```
const [ textAlign, textBaseline ] = evt.target.value.split('-');
```

And then just passing both into the updateTextPosition method below as `{ textAlign, textBaseline }`.

Reply:
Done